es6/README.md

```diff
@@ -704,6 +704,42 @@ Button.propTypes = {
 
 Both declarative property syntaxes are not a part of the ES2015 specification and are a in the midst of the ECMAScript proposal approval process. Currently they are sitting in Stage 3. For details, check out [ECMAScript Class Fields and Static Properties](https://github.com/jeffmo/es-class-fields-and-static-properties).
 
+When defining methods to be attached to event listeners, we recommend using the fat arrow syntax within classes to bind the current context without returning a new function each time.
+
+```js
+
+// good: bind the current context using fat arrow syntax
+class myComponent extends React.PureComponent {
+    onResize = () => {
+        //handle resize window.
+    }
+
+    componentDidMount() {
+        window.addEventListener('resize', this.onResize);
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('resize', this.onResize);
+    }
+}
+
+// bad: use `::func` to bind events need to be removed later
+// the `resize` handler won't be removed as `::onResize` will return a new function
+class myComponent extends React.PureComponent {
+    onResize() {
+        //handle resize window.
+    }
+
+    componentDidMount() {
+        window.addEventListener('resize', ::this.onResize);
```

Comment:
I think @BenAtEventbrite meant we should keep the other `bad` example and get rid of this one since only seat designer uses the bind operator and other places uses `.bind` with `this.handleResize = this._handleResize.bind(this)`

Reply:
hmm, ok, I mean if we do `this.handleResize = this._handleResize.bind(this)` in the constructor then later reference to `this.handleResize` then it is not "too bad" compared to use`::` since it is not introducing bugs. But I feel it's fine if `seat_designer` is the only place having this syntax, and I've started to refactor some of them, hopefully later all will be gone.
Do we want to keep the `::` example just in case people later want to use it for some reasons? 😆 

I will restore the example of `this.handleResize = this._handleResize.bind(this)`

Reply:
updated